zerver/openapi/openapi.py

```diff
@@ -218,6 +218,19 @@ def get_openapi_summary(endpoint: str, method: str) -> str:
     return openapi_spec.openapi()["paths"][endpoint][method.lower()]["summary"]
 
 
+def get_endpoint_from_operationid(operationid: str) -> str:
+    print(operationid)
+    for each_endpoint in openapi_spec.openapi()["paths"]:
+        for each_method in openapi_spec.openapi()["paths"][each_endpoint]:
+            if (
+                "operationId" in openapi_spec.openapi()["paths"][each_endpoint][each_method]
+                and openapi_spec.openapi()["paths"][each_endpoint][each_method]["operationId"]
+                == operationid
```

Comment:
It might be cleaner to do `operationId = openapi_spec.openapi()["paths"][endpoint][method].get("operationId")`

and then just compare that value.